src/include/dfuse_ioctl.h

```diff
@@ -66,9 +67,15 @@ struct dfuse_mem_query {
 	uint64_t pool_count;
 	uint64_t container_count;
 	ino_t    ino;
+	uint32_t stat_count;
 	bool     found;
 };
 
+struct dfuse_stat {
+	uint64_t value;
+	char     name[16];
+};
+
```

Comment:
Since this is in `src/include`, just checking if we forsee any need to extend this struct in the future? Is there a need to version or set aside extra space just in case?

Reply:
There's already DFUSE_IOCTL_VERSION and we can also change the ioctl `cmd` number if we need to change the interface.  This is all new now so we can still change it up until we branch for 2.6 - I expect to continue working on the json output whilst we still have that freedom.